homeassistant/components/switch/doorbird.py

```diff
@@ -2,82 +2,56 @@
 import datetime
 import logging
 
-import voluptuous as vol
-
 from homeassistant.components.doorbird import DOMAIN as DOORBIRD_DOMAIN
-from homeassistant.components.switch import PLATFORM_SCHEMA, SwitchDevice
-from homeassistant.const import CONF_SWITCHES
-import homeassistant.helpers.config_validation as cv
+from homeassistant.components.switch import SwitchDevice
 
 DEPENDENCIES = ['doorbird']
 
 _LOGGER = logging.getLogger(__name__)
 
-SWITCHES = {
-    "open_door": {
-        "name": "Open Door",
-        "icon": {
-            True: "lock-open",
-            False: "lock"
-        },
-        "time": datetime.timedelta(seconds=3)
-    },
-    "open_door_2": {
-        "name": "Open Door 2",
-        "icon": {
-            True: "lock-open",
-            False: "lock"
-        },
-        "time": datetime.timedelta(seconds=3)
-    },
-    "light_on": {
-        "name": "Light On",
-        "icon": {
-            True: "lightbulb-on",
-            False: "lightbulb"
-        },
-        "time": datetime.timedelta(minutes=5)
-    }
-}
-
-PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_SWITCHES, default=[]):
-        vol.All(cv.ensure_list([vol.In(SWITCHES)]))
-})
-
 
 def setup_platform(hass, config, add_devices, discovery_info=None):
     """Set up the DoorBird switch platform."""
     device = hass.data.get(DOORBIRD_DOMAIN)
 
-    switches = []
-    for switch in SWITCHES:
-        _LOGGER.debug("Adding DoorBird switch %s", SWITCHES[switch]["name"])
-        switches.append(DoorBirdSwitch(device, switch))
+    relays = device.info()['RELAYS']
+    relays.append("")  # IR lights
```

Comment:
Why not call it something readable like `'ir_light'`.

Reply:
I used `__ir_light__` because a device is more likely to actually be named `ir_light` if another one is present.